### PR TITLE
Allow creating fulfillments waiting for acceptance

### PR DESCRIPTION
**Pull request by orzechdev**
I want to merge this change because... it allows creating fulfillments waiting for acceptance.
Backend: https://github.com/mirumee/saleor/pull/7675

**PR intended to be tested with API branch:** 

### Screenshots

Zrzut ekranu 2021-07-27 o 14 15 30
Zrzut ekranu 2021-07-29 o 13 12 05

### Pull Request Checklist

1. [x] This code contains UI changes
2. [x] All visible strings are translated with proper context including data-formatting
3. [x] Attributes `[data-test-id]` are added for new elements
4. [x] Changes are mentioned in the changelog
5. [x] The changes are tested in different browsers and in light/dark mode

### Test environment config

API_URI=https://feature-fulfillment-api.api.saleor.rocks/graphql/

### CODE REVIEW

**Comment by patrys** [comment]
Task linked: [SALEOR-3860 Allow creating fulfillments waiting for acceptance](https://app.clickup.com/t/2549495/SALEOR-3860)

**Review by mmarkusik — APPROVED** [review]
✨

**Review by SektorDV — APPROVED** [review]

**Comment by patrys** [comment]
Task linked: [SALEOR-4021 Make fulfillment approvement naming consistent](https://app.clickup.com/t/2549495/SALEOR-4021)

**Review by pwgryglak — APPROVED** [review]

**Comment by orzechdev** [comment]
Update: disable fulfillment creation only when settings 1. auto-approve fulfillments is on and 2. allow fulfillment without payment is off, and when 3. order is not paid:
Zrzut ekranu 2021-07-30 o 14 43 37
Zrzut ekranu 2021-07-30 o 15 24 11

**Review by SektorDV — APPROVED** [review]

File: src/orders/components/OrderFulfillPage/OrderFulfillPage.tsx [hunk 1]
```diff
@@ -351,19 +306,19 @@ const OrderFulfillPage: React.FC<OrderFulfillPageProps> = props => {
...
+                                ?.join(" / ")}
                             </Typography>
                           </TableCellAvatar>
                           <TableCell className={classes.colSku}>
                             {line.variant?.sku}
                           </TableCell>
                           {warehouses?.map(warehouse => {
-                            const warehouseStock = line.variant?.stocks.find(
+                            const warehouseStock = line.variant?.stocks?.find(
                               stock => stock.warehouse.id === warehouse.id
```
Comment (mmarkusik) [inline src/orders/components/OrderFulfillPage/OrderFulfillPage.tsx:322]: getById

**Review by mmarkusik — APPROVED** [review]

**Review by pwgryglak — APPROVED** [review]

File: src/orders/components/OrderFulfillPage/OrderFulfillPage.tsx (hunk 1 above)
Comment (orzechdev) [reply src/orders/components/OrderFulfillPage/OrderFulfillPage.tsx:322]: Stock is taken from warehouse subfield, so getById can't be used here.

**Review by karola312 — APPROVED** [review]

**Pull request merged**